index.md

```diff
@@ -29,25 +29,32 @@ Welcome to the webpage of ISCA's Special Interest Group (SIG) on Robust Speech P
 * [1st CHiME Speech Separation and Recognition Challenge](https://www.isca-speech.org/archive/chime_2011/index.html)
 
 # SIG events
```

Comment:
SIG events should be endorsed by ISCA via RoSP SIG.
Does SASB correspond to that?
If not, we can move it to the related activities.

Reply:
I asked, it is not. Moving it !